editoast/openapi.yaml

```
@@ -2275,6 +2275,13 @@ components:
           type: integer
         train_name:
           type: string
+        train_validation:
+          enum:
+          - Valid
+          - InvalidRollingStock
+          - InvalidRollingStockAndInfra
+          - InvalidInfra
+          type: string
```

```suggestion
        freshness:
          enum:
          - UpToDate
          - NewerRollingStock
          - NewerRollingStockAndInfra
          - NewerInfra
          type: string
```

I got very confused by the naming. The current name suggest that this is related to the validation issue, where the rolling stock and infra would be unusable.